Kentor.AuthServices.StubIdp/Views/Home/Index.cshtml

```diff
@@ -53,4 +53,14 @@
     </div>
 
     <button>Submit</button>
-}
\ No newline at end of file
+}
+
+
+<script>
+    // Focus first relevant field to use
+    if ($("#InResponseTo").val()) {
+        $("#NameId").focus().select();
+    } else {
+        $("#InResponseTo").focus();

```

Comment:
For unsolicited responses (where the login is initiated from the Idp) the InResponseTo should be blank, but in that case the ACS URL can't be detected (as there is no AuthNRequest to extract it from).